src/components/Blurb.vue

```diff
@@ -54,28 +59,31 @@ export default {
                 negativeBias: this.message.issue
                     ? this.message.issue.bias == -1
                     : false,
-                positiveBias: this.message.issue
-                    ? this.message.issue.bias == 1
+                positiveBias: this.message.issue 
+                    ? this.message.issue.bias == 1 
```

Comment:
I appreciate this was my code that I'm criticizing, but we should check for `< 0` and `> 0` instead of `== ± 1` since there's a possibility that the issue will have a non-int value. (we don't enforce that it has to be ±1 anywhere, so it seems reasonable not to impose that assumption on the frontend at least, even if we do eventually decide that it must be an integer.)